decorator.rb

```diff
@@ -0,0 +1,28 @@
+require_relative 'nameable'
+
+class Decorator < Nameable
+  def initialize(nameable)
+    super()
+    @nameable = nameable
+  end
```

Comment:
- [ ] Good job adding the @nameable instance variable. KIndly note that we won't be able to access this variable outside the class if you don't make it accessible by adding `attr_accessor : nameable`. Kindly consider adding it 👍🏽